core/src/main/scala/tapir/generic/Configuration.scala

```diff
@@ -0,0 +1,29 @@
+package tapir.generic
```

Comment:
I'm wondering about the packages ... why `generic` ? Maybe all of this should go to e.g. `schema` package?

Reply:
Ah ok, `generic` because of generic derivation ;)

But then this would be `tapir.schema.generic`? (the "genericness" applies only to the `SchemaFor` typeclass, not the whole project)

Or maybe rename `Configuration` to `SchemaForConfiguration`?

Reply:
On second though, the same kind of configuration will be needed to create a codec which maps form-data to a case class. So `tapic.generic.Configuration` is the right package & name :)

However, `SchemaFor` I think should go back to `tapir`.

Reply:
My point for current layout:
GenericDerivation and its configuration are kept in one place
But if we are going to extend this configuration maybe it should be moved to `tapir` package since it will be more general configuration. After that there is no point in keeping `SchemaFor` in a separate package.

Reply:
So maybe: `SchemaFor` remains in `tapir` (as it's a definition of a typeclass, nothing generic), but  `SchemaForMagnoliaDerivation` goes to `tapir.generic`?